view/frontend/layout/checkout_onepage_success.xml

```diff
@@ -4,8 +4,12 @@
         <referenceContainer name="order.success.additional.info">
             <block class="Omise\Payment\Block\Checkout\Onepage\Success\TescoAdditionalInformation" name="onepage.success.tesco" 
                 template="Omise_Payment::checkout/onepage/success/tesco_additional_info.phtml"/>
-            <block class="Omise\Payment\Block\Checkout\Onepage\Success\PaynowAdditionalInformation" name="onepage.success.paynow" 
+            <block class="Omise\Payment\Block\Checkout\Onepage\Success\OfflineAdditionalInformation" name="onepage.success.paynow" 
```

Comment:
Because Tesco is also an offline payment, however, Tesco payment method does not inherit or using this class. Maybe `OfflineAdditionalInformation` would be a bit confusing for this particular case?

Reply:
Removed `TescoAdditionalInformation` file. Using common file `OfflineAdditionalInformation`